code/game/objects/items/stacks/tiles/tile_reskinning.dm

```diff
@@ -0,0 +1,54 @@
+/////////////////////
+// Tile reskinning //
+/////////////////////
+// Q: What is this?
+// A: A simple function to allow you to change what tiles you place with a stack of tiles.
+// Q: Why do it this way?
+// A: This allows players more freedom to do beautiful-looking builds. Having five types of titanium tile would be clunky as heck.
+// Q: Great! Can I use this for all floors?
+// A: Unfortunately, this does not work on subtypes of plasteel and instead we must change the icon_state of these turfs instead, as the icon_regular_floor var that "saves" what type of floor a plasteel subtype turf was so once repaired...
+// ... it'll go back to the floor it was instead of grey (medical floors turn white even after crowbaring the tile and putting it back). This stops changing turf_type from working.
+
+/obj/item/stack/tile/mineral/titanium/ShiftClick(mob/user)
```

Comment:
Does anything else use shift click like this? I understand that alt-click splits the stack, but I'm not sure if overriding shift click is common.

Reply:
Shift click should just examine, I'm pretty sure it doesn't interfere with anything else.

Reply:
Does self-attack do anything here? Clicking it while in your hand, or pressing Z.
Might be a bit more intuitive than shift click, which is usually for examine instead.

Reply:
Self-attack will bring up the crafting menu for items you craft with titanium floor tiles. There's nothing to actually craft of course.

Reply:
Sounds like you should replace that then :eyes:

Reply:
why not ctrl click when holding